src/tokens.js

```diff
@@ -1132,6 +1132,103 @@ const tokens = {
     return uid === HATHOR_TOKEN_CONFIG.uid;
   },
 
+  /**
+   * Checks if this transaction is the creation of an NFT following the NFT Standard Creation.
+   * @see https://github.com/HathorNetwork/rfcs/blob/master/text/0032-nft-standard.md#transaction-standard
+   * @param {Object} tx Transaction with decoded inputs and outputs
+   */
+  isNFTToken(tx) {
+    // Basic input validation
+    if (!tx) {
+      return false;
+    }
+
+    // If this is not a token creation tx, it cannot be an NFT
+    if (tx.version !== CREATE_TOKEN_TX_VERSION) {
+      return false;
+    }
+
+    // NFT creation must have at least a DataScript output (the first one) and a Token P2PKH output
+    if (tx.outputs.length < 2) {
+      return false;
+    }
+
+    // NFT creation DataScript output must have value 1 and must be of HTR
+    const firstOutput = tx.outputs[0];
+    if (firstOutput?.value !== 1 || firstOutput?.token_data !== 0) {
+      return false;
+    }
+    // First output must also have a standard-compliant script
+    if (!hasStandardScript(firstOutput)) {
+      return false;
+    }
+
+    // Validating the remaining outputs
+    let mintOutputs = 0;
+    let meltOutputs = 0;
+    for (let i = 1; i < tx.outputs.length; ++i) {
+      // All the outputs must also have standard scripts
```

Comment:
This concept of a "standard" script was better discussed in [a comment above](https://github.com/HathorNetwork/hathor-wallet-lib/pull/378#discussion_r898556293) and it was removed from the code to avoid ambiguity.